.gitguardian.example.yml

```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
```

Comment:
Better to have an example with at least two ignored paths to show it is possible